packages/components/CHANGELOG.md

```diff
@@ -340,6 +341,7 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 - `Model Signature`
 - Props `align-header` and `width` for `Carousel`
 - Vue: plugin functions `createPorscheDesignSystem` and `usePorscheDesignSystemPlugin`
+- Prop `theme` for `Table`
```

Comment:
Wrong place, it's currently under section `### [3.0.0-alpha.2] - 2023-02-27`

Reply:
Fixed it